### ISSUE TITLE
Failed validation: #/definitions/QuoteSummaryResult

### ISSUE THREAD

**Issue by LesterLyu** [gadicc/node-yahoo-finance2#20]
## Problem getting quotes for ETFs and mutual funds

- 0P000071W8.TO
- QQQ

```js
const data = await yahooFinance.quoteSummary("0P000071W8.TO");
const data = await yahooFinance.quoteSummary("QQQ");
```

#### Mutual Fund Error:  The following result did not validate with schema: #/definitions/QuoteSummaryResult
```js
[
  {
    keyword: 'required',
    dataPath: '/price',
    schemaPath: '#/definitions/Price/required',
    params: { missingProperty: 'regularMarketDayHigh' },
    message: "should have required property 'regularMarketDayHigh'"
  }
]
```

#### ETF Error:  The following result did not validate with schema: #/definitions/QuoteSummaryResult
```js
[
  {
    keyword: 'additionalProperties',
    dataPath: '/summaryDetail',
    schemaPath: '#/definitions/SummaryDetail/additionalProperties',
    params: { additionalProperty: 'yield' },
    message: 'should NOT have additional properties'
  }
]
```

**Comment by LesterLyu** [comment]
A simple fix would be https://github.com/gadicc/node-yahoo-finance2/issues/16

**Comment by gadicc** [comment]
Hey @LesterLyu, thanks for reporting this.

I published a new version which let's you use all the options to skip validation and be less verbose with errors, as documented at https://github.com/gadicc/node-yahoo-finance2/blob/devel/docs/validation.md.  (This URL is given during the error too).

This was a bit of a rushed release pre-weekend but only in terms of the above, everything else is better tested.  Let me know if you run into trouble.

For the rest of it, will fix the schema hopefully over the weekend.

**Comment by gadicc** [comment]
(talking about 1.5.0)

**Comment by gadicc** [comment]
Thanks @LesterLyu, fixed in 1.5.1 with tests for all quoteSummary submodules for these two symbols.  Please let me know if you have any further issues.  I know these validation issues are annoying in the beta but we'll reap the rewards if we get it right now.  Of course you always still have the option to turn off validation.
